### PR TITLE
PAY-2497:Flux Config for Fees Register Api

### PR DESCRIPTION
**Pull request by amitchidrewar1301**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: k8s/aat/common/ccpay/fees-register-api.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: fees-register-api
+    version: 0.0.7
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "280b07a1-f1a1-4369-8e21-0bb2c2995f1e"
```
Comment (timja) [inline k8s/aat/common/ccpay/fees-register-api.yaml]: shouldn't this be coming from vault?

File: k8s/prod/common/ccpay/fees-register-api.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (timja) [inline k8s/prod/common/ccpay/fees-register-api.yaml]: this isn't approved for prod yet, please get it deployed to AAT and then we can get it reviewed by SecOps

File: k8s/aat/common/ccpay/fees-register-api.yaml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (timja) [inline k8s/aat/common/ccpay/fees-register-api.yaml]: Missing codeowners changes

File: k8s/aat/common/ccpay/fees-register-api.yaml (hunk 1 above)
Comment (amitchidrewar1301) [reply k8s/aat/common/ccpay/fees-register-api.yaml]: Yes, so removed entry

File: k8s/prod/common/ccpay/fees-register-api.yaml (hunk 2 above)
Comment (amitchidrewar1301) [reply k8s/prod/common/ccpay/fees-register-api.yaml]: for timebeing removed config for production

File: k8s/aat/common/ccpay/fees-register-api.yaml (hunk 3 above)
Comment (amitchidrewar1301) [reply k8s/aat/common/ccpay/fees-register-api.yaml]: Added Code owner entry

**Pull request merged**